cmd/amppkg_dl_sxg/main.go

```diff
@@ -90,11 +99,25 @@ func main() {
 	if err != nil {
 		log.Fatalf("%+v", err)
 	}
+	cURL, err := url.Parse(certURL)
+	if err != nil {
+		log.Fatalf("%+v", err)
+	}
+	fURL, err := url.Parse(*flagCertURL)
+	if err != nil {
+		log.Fatalf("%+v", err)
+	}
+	if *flagCertURL != "" {
+		cURL.Scheme = fURL.Scheme
```

Comment:
How about something like:

```go
certHash := path.Base(cURL.Path)
cURL = fURL
cURL.Path = path.Join(cURL.Path, certHash)
```

That way, the user can provide a `user:pass@` or `?query` too. (Admittedly, rare cases.)

Reply:
Fixed.